### ISSUE TITLE
Core 3.0 compatibility

### ISSUE THREAD

**Issue by alexandrejobin** [DevelAx/BootstrapCore#1]
Hi!

I have used your trick to override the default styling of the error messages but now that I'm converting my project to Core 3.0, it doesn't seems to work anymore. I get the error message 

> System.FieldAccessException: 'Cannot set initonly static field 'ValidationInputValidCssClassName' after type 'Microsoft.AspNetCore.Mvc.ViewFeatures.HtmlHelper' is initialized.'

Do you have any idea how to solve this?

**Comment by bigcitydev** [comment]
Hello my friend, do you found any solution for this problem??

**Comment by alexandrejobin** [comment]
Nope. I haven't found any solution. I've decided to declare overrides inside my scss for now.
